Frameworks/Foundation/NSIndexPath.mm

```diff
@@ -38,22 +49,36 @@ + (NSIndexPath*)indexPathWithIndexes:(const NSUInteger*)indexes length:(NSUInteg
  @Status Interoperable
 */
 + (NSIndexPath*)indexPathForRow:(NSInteger)row inSection:(NSInteger)section {
-    unsigned indexes[] = { section, row };
+    if (row < 0 || section < 0) {
```

Comment:
The reference platform hasn't yet implemented this so I cannot say...
So long as row >= 0 && section >= 0 NSUInteger can hold it